Content.Server/Procedural/DungeonJob/DungeonJob.EntityTableDunGen.cs

```diff
@@ -36,7 +37,7 @@ private async Task PostGen(
                 if (!ValidateResume())
                     return;
 
-                if (reservedTiles.Contains(tile))
+                if (reservedTiles.Contains(tile) && gen.Table is not NestedSelector)
                     continue;
```

Comment:
Isn't the issue that it can't generate full groups due to how many ores there are?

Wouldn't it be better to target that rather than adding a hidden behavior?

Reply:
> Isn't the issue that it can't generate full groups due to how many ores there are?

I don't think so, if I completely remove everything from the VGRoid spawner except "VGRoidExterior" items still won't spawn. I'm fairly certain it's because every tile of the dungeon is getting marked as reserved preventing items from spawning? I might be wrong but that's what it seems like to me.

Would adding a ignoreReserved bool to EntityTableDunGen be acceptable?